src/Traits/JsonSerializer.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Softonic\GraphQL\Traits;
+
+trait JsonSerializer
+{
+    public function jsonSerialize(): array
+    {
+        $item = [];
+        if ($this->hasChanged()) {
+            foreach ($this->arguments as $key => $value) {
```

Comment:
Duplicated code! It is better to use the already implemented `toArray` method

Reply:
Tried but we realised that jsonSerialize() is different for Item and Mutation and cannot be unified.